### ISSUE TITLE
Red alert: Small GUI bug in "Tanya" build button

### ISSUE THREAD

**Issue by Diranus** [OpenRA/OpenRA#14263]
Hello,

here is a small bug in the GUI while gameplay:

You can build a tanya only once. After that, the button to build a tanya is greyed out. Everything OK so far.

Now, put the tanya into a ranger (or something else that can carry infantery units in it)

Now, while the tanya is inside the unit, the tanya button isn't greyed out anymore while it is still impossible to build a second one. It should keep greyed out regardless the tanya is inside another unit or not^^.

**Comment by GraionDilach** [comment]
Duplicate of #5872.

**Comment by pchote** [comment]
Closing as duplicate.
